language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<p>Could you test this in Vertex AI Workbench as well? I believe this may not work directly in Workbench (b/c it uses a service account which may not have sufficient permission to add IAM policy bindings. So the code may need to be run outside of Workbench, such as in Cloud Shell. If so, it would be important to add in some instructions on how to run these gcloud commands in Cloud Shell).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Good shout thank you! I tested this and it failed as expected. I've added the below text. Please shout if you know of another notebook that has a good example of this:</p><p> </p><p><strong>Note:</strong>&nbsp;If you are using Vertex AI workbench, the service account used by Vertex AI may not have sufficient permissions to add IAM policy bindings.</p><p>The&nbsp;<a href="https://colab.research.google.com/corgiredirector?site=https%3A%2F%2Fcloud.google.com%2Fiam%2Fdocs%2Fgranting-changing-revoking-access%23grant-single-role&amp;link_redirector=1" target="_blank">IAM Grant Access</a>&nbsp;page gives instructions on how these policy bindings can be added using Cloud Shell.</p>